aws-athena/views/default-vw_pin_sale.sql

```diff
@@ -235,9 +232,15 @@ max_version_flag AS (
 sales_val AS (
     SELECT
         sf.meta_sale_document_num,
-        sf.sv_is_outlier,
-        sf.sv_is_ptax_outlier,
-        sf.sv_is_heuristic_outlier,
+        CASE WHEN sf.sv_is_outlier = 1 THEN TRUE
+            WHEN sf.sv_is_outlier = 0 THEN FALSE
+        END AS sv_is_outlier,
+        CASE WHEN sf.sv_is_ptax_outlier = 1 THEN TRUE
+            WHEN sf.sv_is_ptax_outlier = 0 THEN FALSE
+        END AS sv_is_ptax_outlier,
+        CASE WHEN sf.sv_is_heuristic_outlier = 1 THEN TRUE
+            WHEN sf.sv_is_heuristic_outlier = 0 THEN FALSE
+        END AS sv_is_heuristic_outlier,
```

Comment:
@wagnerlmichael Just merged a PR to fix this in the underlying tables, so this should be redundant. Let's revert it to the previous hunk.